rero/rero-ils
# Libraries: add options opening_hours + new library editor

* Add opening hours on json schema
* Library and Opening Hours editor
* My Library link with redirect to editor

To Test:
- ./script/bootstrap
- ./script/setup
- Check link "My Library"
- Library: Check link "Edit" on library
- Save data and show new values on complete view.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>
Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
